### PR TITLE
chore: Fix linting errors and warnings

### PR DESCRIPTION
**Pull request by ovineq**
Goal is to remove `<any>` from the code, fix the linting errors and make linting part of the CI.
Issue: https://github.com/equinor/esv-intersection/issues/489

### CODE REVIEW

File: src/control/ExtendedCurveInterpolator.ts [hunk 1]
```diff
@@ -148,8 +148,8 @@ export class ExtendedCurveInterpolator extends CurveInterpolator {
     return this.getPointAt(t);
   }
 
-  getPointAt(t: number): Vector {
+  getPointAt(t: number): number[] {
     const tl = clamp(t, 0, 1);
-    return super.getPointAt(tl);
+    return super.getPointAt(tl) as number[];
```
Comment (schoening) [inline src/control/ExtendedCurveInterpolator.ts:153]: Is casting is necessary here? I think we try to avoid casting generally.

**Review by schoening — APPROVED** [review]
Veldig bra jobbet :) !
Dette kommer til å spare oss masse tid!

**Comment by schoening** [comment]
image
ESLint error `LayerManager.ts` line `199`

**Review by ooystein — APPROVED** [review]
Looking forward to this!

**Pull request merged**